Python/Gmeet py Bot/Gmeet_buddy.py

```diff
@@ -83,12 +91,13 @@ def join_classes():
     time.sleep(5)
     keyboard.press_and_release('Win+alt+r')
     print("meeting Recording Started")
-    
+
+
 join_classes()
 
-#schedule according to your time table and quit the browser according to the time of completion of your class
+# schedule according to your time table and quit the browser according to the time of completion of your class
 # schedule.every().day.at("09:00").do(join_classes)
```

Comment:
This is a code block right? Just commended out because is situational, if that's the case.

Please put it in triple quoted strings
```
"""
Like This
"""
```
(You can ensure indentation this way)